packages/lesswrong/components/common/Header.tsx

```diff
@@ -69,6 +69,7 @@ export const styles = (theme: ThemeType): JssStyles => ({
     top: 3,
     paddingRight: theme.spacing.unit,
     color: theme.palette.text.secondary,
+    maxWidth: 130,
```

Comment:
Is this necessary? I am worried this will cause a linebreak on pages with long page-titles

Reply:
probably not necessary, let's discuss tomorrow a better way to do it